### ISSUE TITLE
Parallelize Indexing

### ISSUE THREAD

**Issue by reglim** [docat-org/docat#405]
It's too slow :( 

It might not be possible, but it'd be really nice if index_all_projects could for example create a temporary database for each project, let all of those projects be indexed  in parallel and combine the results again

**Comment by reglim** [comment]
As we don't do full-text search anymore, this is no longer relevant.
